stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/dto/DownstreamMessageDto.kt

```diff
@@ -0,0 +1,20 @@
+package io.getstream.chat.android.client.api2.model.dto
+
+import com.squareup.moshi.JsonClass
+import java.util.Date
+
+@JsonClass(generateAdapter = true)
+internal data class DownstreamMessageDto(
```

Comment:
In the old `Message` class, we have many `@IgnoreSerialisation` annotations. Here, we'll use separate models for `Message` objects being sent and received instead, so that there are no always-unused fields to deal with.